src/main/scala/fs2/aws/kinesis/KinesisWorkerSettings.scala

```diff
@@ -0,0 +1,31 @@
+package fs2
+package aws
+package kinesis
+
+import scala.concurrent.duration._
+
+case class KinesisWorkerStreamSettings(
```

Comment:
Rename the file "KinesisSettings" ? there are all your Settings models here not only "KinesisWorkerSettings"